tests/bearlib/languages/LanguageTest.py

```diff
@@ -0,0 +1,21 @@
+import unittest
+
+from coalib.bearlib.languages.Language import Language
+
+
+class LanguageAttributeErrorTest(unittest.TestCase):
+
+    def setUp(self):
+        self.lang_cpp = Language['CPP']
+        self.lang_unknown = Language['Unknown']
+
+    def tearDown(self):
+        pass
+
+    def test_invalid_attribute(self):
+        with self.assertRaises(AttributeError):
+            self.lang_cpp.not_an_attribute
+
+    def test_attribute_list_empy(self):
+        with self.assertRaises(AttributeError):
```

Comment:
Use assertRaisesRegexp with a short part of the message to verify you have the correct message